images/executor/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12-alpine AS builder
```

Comment:
Cool use of multi-stage builds!

Is there a reason to use `node:12-alpine` as the builder, rather than another copy of `centos7-plbase`? I’m thinking that the latter would ensure that we build with exactly the same node version and environment.

Reply:
I think this is a remnant of an earlier iteration where I was experimenting with building a slimmer executor image not based on `prairielearn/centos7-plbase`. I'll change this as you described!